openapi/src/test/java/io/helidon/openapi/TestUtil.java

```diff
@@ -221,6 +222,12 @@ public static JsonStructure jsonFromResponse(HttpURLConnection cnx) throws IOExc
         return result;
     }
 
+    static JsonStructure jsonFromReader(Reader reader) {
+        JsonReader jsonReader = JSON_READER_FACTORY.createReader(reader);
+        JsonStructure result = jsonReader.read();
```

Comment:
Nit: any kind of `close` operation that needs to be performed here?

Reply:
Yes. Done.